src/testdata/redirect_data/b.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["file:///b.ts"],"sourcesContent":["export const b = \"b\";\n"],"names":[],"mappings":"AAAA,MAAM,CAAC,KAAK,CAAC,CAAC,GAAG,CAAG"}
```

Comment:
Swc changed its sourcemap output (which looks valid to me, but has more detail. I went through and looked at the changes). For this reason, I stored the old sourcemap output to compare to the existing eszip files in separate files here.